src/routes/repetition/components/Chart.tsx

```diff
@@ -0,0 +1,68 @@
+import { useFocusEffect } from '@react-navigation/native'
+import React, { ReactElement, useCallback, useEffect, useState } from 'react'
+import { heightPercentageToDP as hp } from 'react-native-responsive-screen'
+import { LineSegment, VictoryAxis, VictoryBar, VictoryChart, VictoryLabel } from 'victory-native'
+
+import theme from '../../../constants/theme'
+import useLoadAsync from '../../../hooks/useLoadAsync'
+import { getNumberOfWordsInEachSection, sections } from '../../../services/RepetitionService'
+import { getLabels } from '../../../services/helpers'
+
+const { chartColor1, chartColor2, chartColor3, chartColor4 } = theme.colors
+const barColors = [chartColor1, chartColor2, chartColor2, chartColor2, chartColor2, chartColor2, chartColor3]
+const Chart: () => ReactElement = () => {
```

Comment:
Optional: If this is repetition specific this should also have a repetition specific name imo. While using this, one does not get what kind of chart this :see_no_evil: